python/CMakeLists.txt

```diff
@@ -62,9 +62,11 @@ target_link_libraries(_pycudaq
 if (NOT SKBUILD)
   install(DIRECTORY cudaq DESTINATION .)
   install(TARGETS _pycudaq DESTINATION cudaq)
-  # Also install these to the build directory for testing.
-  install(DIRECTORY cudaq DESTINATION ${CMAKE_BINARY_DIR}/python)
-  install(TARGETS _pycudaq DESTINATION ${CMAKE_BINARY_DIR}/python/cudaq)
+  # Also move these to the build directory for testing.
```

Comment:
How does this custom command work? And I'm guessing the issue was from installing to the build instead of copying?